docs/process/guidance/feature_flags/index.rst

```diff
@@ -0,0 +1,507 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2024 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Feature Flags
```

Comment:
Aren't "feature flags" confusing, when we have "features".
Since "feature flags" and "features" are totally unrelated to my understanding?

Reply:
Do we really want to give this technique a different name than all the bazel community is using? I would hesitate a lot...

Also feature flags are not unrelated from features. We enable and configure features and their subfeatures with the flags.

Reply:
I cannot resolve my own comments here... the button is gone